cmd/main.go

```diff
@@ -235,8 +235,11 @@ func watchConfigFile(filename, adapterName string) {
 		return false
 	}
 
+	ticker := time.NewTicker(1 * time.Second)
+	defer ticker.Stop()
+
 	// begin poller
-	for range time.Tick(1 * time.Second) {
+	for range ticker.C {
```

Comment:
Yeah... I flip-flopped here. This was from [SA1015](https://staticcheck.io/docs/checks#SA1015):

```
SA1015: using time.Tick leaks the underlying ticker, consider using it only in endless functions, tests and the main package, and use time.NewTicker here (staticcheck)
```

The reason it's triggering is because this isn't the `main` package. I'll ignore this one instead.